plenigo_plugin/PlenigoShortcodeManager.php

```diff
@@ -84,6 +121,9 @@ function plenigo_register_tinymce_js($plugin_array) {
         $plugin_array['plenigo_renew'] = plugins_url('../plenigo_js/tinymce-plenigo_renew-plugin.js', __file__);
         $plugin_array['plenigo_failed'] = plugins_url('../plenigo_js/tinymce-plenigo_failed-plugin.js', __file__);
         $plugin_array['plenigo_separator'] = plugins_url('../plenigo_js/tinymce-plenigo_separator-plugin.js', __file__);
+        // TODO add missing JS files
```

Comment:
Totally, in fact I have take them out in this branch but got re-added when you pulled master into this branch again.